azure_functions_worker/bindings/__init__.py

```diff
@@ -1,18 +1,22 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-from .tracecontext import TraceContext
-from .retrycontext import RetryContext
+from .retrycontext import RetryContext  # isort: skip
+from .tracecontext import TraceContext  # isort: skip
```

Comment:
Skipping these two for preventing cyclic import issue when importing `from .context import Context`